### ISSUE TITLE
Training Memory Consumption

### ISSUE THREAD

**Issue by wondervictor** [facebookresearch/maskrcnn-benchmark#528]
## ❓ Questions and Help

When training MaskR-CNN with ResNet-101-FPN, I found that memory is only 6.087 GB from training log which is less than that (7.9GB) provided in [MODEL_ZOO](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/MODEL_ZOO.md). Memory information in the training log is obtained by [`torch.cuda.max_memory_allocated() / 1024.0 / 1024.0`](https://github.com/facebookresearch/maskrcnn-benchmark/blob/4a21d69f6db59de57e57da766ffb843448cf542c/maskrcnn_benchmark/engine/trainer.py#L101).

I wonder what causes this difference ?

**Comment by fmassa** [comment]
Hi,

I have seen reports mentioning the same thing.

There seems to have been an improvements in the memory consumption of PyTorch in a recent version. This might be related to using a newer version of CUDNN.

It might be good to re-run all the experiments using latest PyTorch / CUDNN to update the table in the MODEL_ZOO.

The memory savings are pretty significant, which is great! :-)

**Comment by ShihuaiXu** [comment]
Are u using the same dataset with the MODEL_ZOO? I put my own dataset in Mask R-CNN and the menory is different, it is normal?
